docs/platform-openrpc.json

```diff
@@ -0,0 +1,1208 @@
+                },
+                {
+                    "$ref": "#/components/errors/AccountSelectionNoMatch"
+                }
+            ]
+        },
+        {
+            "name": "ledgerlive.account.sync",
+            "summary": "Synchronize the account with its network",
+            "description": "Synchronize the account with its network",
+            "paramStructure": "by-name",
+            "tags": [
+                {
+                    "$ref": "#/components/tags/Accounts"
+                },
+                {
+                    "$ref": "#/components/tags/PermissionReadAccount"
+                }
+            ],
+            "params": [
+                {
+                    "name": "account_id",
+                    "summary": "The account ID of the account to synchronize",
+                    "description": "The account ID of the account to synchronize",
+                    "required": true,
+                    "schema": {
+                        "$ref": "#/components/schemas/AccountId"
+                    }
+                }
+            ],
+            "result": {
+                "name": "account",
+                "description": "An up to date version of the account object",
+                "summary": "An up to date version of the account object",
+                "required": true,
+                "schema": {
+                    "$ref": "#/components/schemas/Account"
+                }
+            },
+            "errors": [
+                {
+                    "$ref": "#/components/errors/PermissionDenied"
+                },
+                {
+                    "$ref": "#/components/errors/AccountIdDoesNotExist"
+                }
+            ]
+        },
+        {
+            "name": "ledgerlive.account.receive",
+            "summary": "Get a reception address for the account and optionally trigger address verification on the device.",
+            "description": "Get a reception address for the account and optionally trigger address verification on the device.",
+            "paramStructure": "by-name",
+            "tags": [
+                {
+                    "$ref": "#/components/tags/Accounts"
+                },
+                {
+                    "$ref": "#/components/tags/PermissionReadAccount"
+                }
+            ],
+            "params": [
+                {
+                    "name": "account_id",
+                    "required": true,
+                    "schema": {
+                        "$ref": "#/components/schemas/AccountId"
+                    }
+                },
+                {
+                    "name": "verify",
+                    "summary": "Set to true, to verify the address on device",
+                    "required": false,
+                    "schema": {
+                        "type": "boolean"
+                    }
+                }
+            ],
+            "result": {
+                "name": "address",
```

Comment:
Would be amazing to have a signed by ledger version of this address in the long term